susemanager/src/mgr_bootstrap_data.py

```diff
@@ -328,7 +328,7 @@
     "python3-msgpack",
     "python3-psutil",
     "python3-py",
-    "python3-pycrypto",
+    "python3-M2Crypto",
```

Comment:
I would suggest to use the either/or syntax. Would that copy both if available ?
```suggestion
    "python3-pycrypto|python3-M2Crypto",
```

Reply:
No, only the first one found. But why do you want both? `salt.spec` clearly specifies that requires `python3-M2Crypto` for any SLE15 version.

Reply:
15 GA does not have it. Only with LTSS it is available. When we copy the old one as well, a customer could at least install an older version of salt.

Reply:
15GA is not supposed to be used anymore, as it's EoL, but OK. I will add it.

Reply:
Done.